x/distribution/keeper/params.go

```diff
@@ -7,37 +7,47 @@ import (
 
 // GetParams returns the total set of distribution parameters.
 func (k Keeper) GetParams(clientCtx sdk.Context) (params types.Params) {
-	k.paramSpace.GetParamSet(clientCtx, &params)
+	store := clientCtx.KVStore(k.storeKey)
+	bz := store.Get(types.ParamsKey)
+	if bz == nil {
+		return params
+	}
+
+	k.cdc.MustUnmarshal(bz, &params)
 	return params
 }
 
 // SetParams sets the distribution parameters to the param space.
-func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
-	k.paramSpace.SetParamSet(ctx, &params)
+func (k Keeper) SetParams(ctx sdk.Context, params types.Params) error {
```

Comment:
i'd say let's add an API-breaking change on the keeper method?

Reply:
WDYM exactly? The API doesn't change -- only the storage and management mechanics.

Reply:
there's a new `error` return arg